src/classes/CON_Email_TEST.cls

```diff
@@ -55,6 +55,24 @@ private class CON_Email_TEST {
         insert listCon;
 	}
 
+	@isTest static void testSingleEmailSmartSet() {
+
+		Contact newCont = new Contact(
+			FirstName = 'JohnnyTest',
+			LastName = 'JohnnyTest',
+			WorkEmail__c = 'workit@test.com'
+		);
+
+		Test.startTest();
+        insert newCont;
```

Comment:
Is the process exactly the same for converting a lead? Should we consider testing also using the Database.convertLead method, since that was the original issue?